### ISSUE TITLE
Add support for downloading SDKs

### ISSUE THREAD

**Issue by j-f1** [XcodesOrg/XcodesApp#394]
Xcode 15 now only includes the macOS SDK by default. You can then launch the app to download additional simulators/SDKs, but only after Xcode is fully installed & launched.

I would like to see the available SDKs listed somewhere either in the inspector or as a collapsable area under each Xcode, allowing them to be downloaded & uncompressed in parallel with Xcode. Since Xcode apparently looks for SDKs in the Downloads folder on first launch, it may be enough to drop the SDK downloads there. Otherwise, they would need to be manually installed somehow.

**Comment by auroralia** [comment]
agreed! the current installation (through apple website) is so slow and randomly stops but through Xcode, it's way faster! we deffo need this, especially as dev builds are moving so fast.

**Comment by lewis-smith** [comment]
Possible duplicate: https://github.com/XcodesOrg/XcodesApp/issues/298
